machida/lib/wallaroo/__init__.py

```diff
@@ -69,13 +72,22 @@ def build_application(app_name, pipeline):
     return pipeline.__to_tuple__(app_name)
 
 
+def range_windows(wrange):
+    return RangeWindowsBuilder(wrange)
+
+
+def count_windows(wrange):
```

Comment:
Would it be simpler for users to call `CountWindowsBuilder(...)` directly?

In which case, could it be renamed `class CountWindows`?